### PR TITLE
Homework1-Andrey

### PR DESCRIPTION
**Pull request by Ad04a**

### CODE REVIEW

File: homework/12_B/03_Andrey/homework1/test_circles.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    #2 point 2 and 3 are NOT equal
+    assert not (Point2 == Point3)
+
+def test_circle_eq():
+    circle1 = Circle(Point(0,0),0)
+    circle2 = Circle(Point(0,0),0)
+    circle3 = Circle(Point(0,0),1)
+    
+    #1 circle 1 and 2 are equal
+    assert (circle1 == circle2)
```
Comment (svetlin-mladenov) [inline homework/12_B/03_Andrey/homework1/test_circles.py:19]: You don't need the () after the assert.

File: homework/12_B/03_Andrey/homework1/test_circles.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    assert (check_circles(circle4, circle5) is CircleValues.INTERCEPTING)
+    assert (check_circles(circle5, circle4) is CircleValues.INTERCEPTING)
+
+    #11,12 circle 5 intercepts with circle 2 from within
+    assert (check_circles(circle5, circle2) is CircleValues.INTERCEPTING)
+    assert (check_circles(circle2, circle5) is CircleValues.INTERCEPTING)
+
+    #12,13 circle 7 touches circle 2 from within
+    assert (check_circles(circle7, circle2) is CircleValues.TOUCHING)
+    assert (check_circles(circle2, circle7) is CircleValues.TOUCHING)
```
Comment (svetlin-mladenov) [inline homework/12_B/03_Andrey/homework1/test_circles.py:58]: I might be wrong but I think there might be a bug in your code: can you add the following test conditions
```
assert check_circles(Circle(Point(0,0), 10), Circle(Point(9,0), 1)) is CircleValues.TOUCHING
```

**Review by svetlin-mladenov — CHANGES_REQUESTED** [review]

File: homework/12_B/03_Andrey/homework1/test_circles.py (hunk 2 above)
Comment (Ad04a) [reply homework/12_B/03_Andrey/homework1/test_circles.py]: FIxed

File: homework/12_B/03_Andrey/homework1/test_circles.py (hunk 1 above)
Comment (Ad04a) [reply homework/12_B/03_Andrey/homework1/test_circles.py]: Fixed

**Pull request merged**